### ISSUE TITLE
Sometimes the gce instances are destroyed

### ISSUE THREAD

**Issue by tomwilkie** [weaveworks/weave#791]
https://circleci.com/gh/weaveworks/weave/464
```
+ gcloud compute instances delete --zone us-central1-a -q host2-464-0 host1-464-0 command cd $SRCDIR/test; ./gce.sh destroy took more than 10 minutes since last output
```

https://circleci.com/gh/weaveworks/weave/488 - test timedout

**Comment by rade** [comment]
This was fixed by #797.

**Comment by tomwilkie** [comment]
Not 100% fixed. If you cancel a run they don't get cleaned up. Going to
write a gc for it one day.
